composer.json

```diff
@@ -128,6 +128,19 @@
                     "type": "zip"
                 }
             }
+        },
+        {
+            "type": "package",
+            "package": {
+                "name": "drupal/entity_usage_queue_tracking",
+                "type": "drupal-module",
+                "version": "9.1.x-dev",
+                "source": {
+                    "type": "git",
+                    "url": "https://git.drupalcode.org/project/entity_usage_queue_tracking.git",
+                    "reference": "9290c86e9c1ed10a3f976c9090c7108e8fcc487e"
+                }
+            }
```

Comment:
The contrib module wasn't pulling down, and I'm not entirely certain why. It should be meeting the criteria specified here: https://www.drupal.org/docs/develop/managing-a-drupalorg-theme-module-or-distribution-project/creating-a-new-project/general-projects